rust_src/crates/webrender/src/output.rs

```diff
@@ -464,7 +636,16 @@ impl Output {
         txn.set_document_view(device_rect);
         self.render_api.send_transaction(self.document_id, txn);
 
-        self.window_context.resize(size.clone());
+        self.webrender_surfman
+            .resize(Size2D::new(size.width as i32, size.height as i32))
+            .unwrap();
+    }
+
+    pub fn deinit(self) {
```

Comment:
use rust drop trait here, the output will be deinit in `destory_frame` automatically.

https://doc.rust-lang.org/std/ops/trait.Drop.html